packages/db-migrations/databases/fxa/patches/patch-164-165.sql

```diff
@@ -0,0 +1,101 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('164');
+
+-- Add verificationMethod to password forgot token table and account reset token table
+ALTER TABLE `passwordForgotTokens` ADD COLUMN `verificationMethod` INT DEFAULT NULL, ALGORITHM = INPLACE, LOCK = NONE;
```

Comment:
I don't believe we should have an issue doing these migrations since these tables are not as big as account or session table.

Reply:
`INSTANT` instead of `INPLACE`?